src/core/shards/core.hpp

```diff
@@ -1421,6 +1427,14 @@ struct Sequence : public SeqBase {
   Types _seqTypes{};
   std::deque<Types> _innerTypes;
 
+  static SHOptionalString help() { return SHCCSTR("Creates an empty sequence (or table if a key is passed)."); }
+
+  static SHTypesInfo inputTypes() { return CoreInfo::NoneType; }
```

Comment:
Why override `inputTypes()` and `outputTypes()`? This isn't correct.

Reply:
Fixed (changes rebased and force-pushed to this commit).